aws/policy/security-services.yaml

```
@@ -204,3 +197,4 @@ Statement:
           - 'eks-nodegroup.amazonaws.com'
           - 'transitgateway.amazonaws.com'
           - 'network-firewall.amazonaws.com'
+          - 'ecs.amazonaws.com'
```

In addition to the change @gravesm suggested, we also need:
```suggestion
          - 'ecs.amazonaws.com'
          - 'ecs-test.amazonaws.com'
```
This will push the file over the character limit even with your wildcard changes, #266 will help with that.